### DISCUSSION TITLE
2 basic scanning questions

### DISCUSSION THREAD

**Discussion by giluriel** [DSheirer/sdrtrunk#2099] [Q&A]
Hi,
Playing around with the software (love it) - I'm missing two very basic things:
1. When I have a playlist with multiple frequencies and the channel locks to one of them - Can I "tell" it to skip to the next freq. even thou it is locked? (Just like pressing SKIP on a physical scanner?
2. Is there a way to manually enter a frequency and set decoding - straight into a channel without saving it in the playlist?

TNX

**Comment by noamlivne** [comment]
Hi,
Perhaps others will have different answers, but here are mine:
1. AFAIK, there is no such option in SDRTrunk. The best solution, IMHO, is to Record all the Aliases/Talkgroups and let Rdio-Scanner (https://github.com/chuot/rdio-scanner) act as your "scanner" (plus listen to almost live and historically via your phone).

2. AFAIK, there is no such option in SDRTrunk. Till this option will be available, the best solution, IMHO, is to use DSDPlus Fastlane ($25USD) for that feature and much more, as a separate "scanner".

Best Regards,
Noam
